### DISCUSSION TITLE
Encryption for data at rest

### DISCUSSION THREAD

**Discussion by gromain** [immich-app/immich#359] [Q&A] [answered]
Hi,

I did not see anything about any kind of encryption for the data Immich stores.
Is there any?

I'm looking for a solution where the users could store their pictures on the service but their admin could not access them.

Thanks for the good work though! Keep it up!

Cheers,
Romain

**Comment by alextran1502** [comment, marked as answer]
Hi,

We might implement this feature at some points in the future. Currently we are focusing on building out the core functionality of the "backup", "view", "shared" and "organized" features.

Can you please put in the feature request for this so we don't forget about it?

Thanks
